packages/nextra/src/mdx-plugins/remark-headings.ts

```diff
@@ -46,7 +46,33 @@ export const remarkHeadings: Plugin<[], Root> = function (this: Processor) {
             id:
               (node.data?.hProperties as HProperties)?.id || slugger.slug(value)
```

Comment:
```suggestion
              slugger.slug((node.data?.hProperties as HProperties)?.id || value)
```
Thanks for your report and PR, but we already use github-slugger package which is responsible for adding suffixes for duplicated headings, I think it can be fixed by the above suggestions. Could you try it?

Also, it should be applicable to h1, since user can have multiple h1 in his page, even it’s not a best practice

Reply:
Oh, I didn't noticed the description of the slugger function, seems nice and convenient, I removed the code and made the suggested change, thanks for the help